src/api/views.py

```diff
@@ -8,6 +8,11 @@
 from django.utils import timezone
```

Comment:
The import `from django.utils import timezone` is not used in the current code and can be removed to clean up the file.